packages/effects-core/src/engine.ts

```
@@ -162,104 +209,243 @@ export class Engine implements Disposable {
+    if (skipRender) {
+      this.onRenderError?.(new Error('Play when texture offloaded.'));
+
+      return this.ticker?.pause();
+    }
+    this.renderer.setFramebuffer(null);
+    this.renderer.clear({
+      stencilAction: TextureLoadAction.clear,
+      clearStencil: 0,
+      depthAction: TextureLoadAction.clear,
+      clearDepth: 1,
+      colorAction: TextureLoadAction.clear,
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)